File: baekjoon/4344/4344.cpp
/*
평균은 넘겠지 C++ 풀이
https://doctcoder.tistory.com/entry/%ED%8F%89%EA%B7%A0%EC%9D%80-%EB%84%98%EA%B2%A0%EC%A7%80-%ED%92%80%EC%9D%B4
*/
#include <iostream>

using namespace std;
int main()
{
	int c,num;
	cin>>c;
	int arr[1000]={0,};
	double mean;
	for(int i=0;i<c;i++)
	{
		cin>>num; // 학생 수
		int sum=0;
		for(int j=0;j<num;j++)
		{
			cin>>arr[j];
			sum+=arr[j];
		}
		mean=(double)sum/(double)num; // 평균
		int count=0;
		for(int j=0;j<num;j++)
		{
			if(arr[j]>mean) count++;
		}
		double res= (double)count/(double)num*100; // 비율
		cout<<fixed; // 소수점 셋째 자리 반올림
		cout.precision(3);
		cout<<res<<"%"<<"\n";
	}
}

File: baekjoon/4344/4344.py
'''
평균은 넘겠지 python 풀이
https://doctcoder.tistory.com/entry/%ED%8F%89%EA%B7%A0%EC%9D%80-%EB%84%98%EA%B2%A0%EC%A7%80-%ED%92%80%EC%9D%B4
'''
c=int(input())
for i in range(c):
	sre=list(map(int,input().split()))
	mean=0
	n=sre[0]
	del sre[0]
	for j in range(n):
		mean+=sre[j]
	mean/=n
	per=0
	for j in range(n):
		if(sre[j]>mean): per+=1
	per=per/n*100
	print(f'{round(per,3):.3f}%')
